### DISCUSSION TITLE
Why does TaskList generate a disabled checkbox?

### DISCUSSION THREAD

**Discussion by fkoji** [thephpleague/commonmark#640] [Q&A] [answered]
## Question

The `<input type="checkbox">` elements converted `- [ ]` using TaskList extension have a disabled attribute. Is this an expected behavior?

<img width="429" alt="スクリーンショット 2020-11-24 16 32 14" src="https://user-images.githubusercontent.com/96885/100062835-9c1a0600-2e73-11eb-825b-74983eb55604.png">

**Comment by colinodell** [comment, marked as answer]
Yes, this behavior is expected and intentional: https://github.com/thephpleague/commonmark/blob/1.5.7/src/Extension/TaskList/TaskListItemMarkerRenderer.php#L39

This aligns with other Markdown parsers (like Markdig) which have the ability to render checkboxes but don't offer functionality to update the Markdown contents when the checkboxes are clicked.  Once #511 is implemented I'd feel comfortable making these non-disabled by default.

**Comment by fkoji** [comment]
Thank you for your reply. For now, I'm going to use JavaScript to update the attributes.
